expfmt/encode_test.go

```diff
@@ -18,8 +18,8 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/golang/protobuf/proto" //nolint:staticcheck // Ignore SA1019. Need to keep deprecated package for compatibility.
 	dto "github.com/prometheus/client_model/go"
+	"google.golang.org/protobuf/proto"
```

Comment:
You could fix the block structure here (it wasn't quite right before, see other nitpicking comments).